Docker/gen.validators.sh

```diff
@@ -120,7 +120,7 @@ cat > ${target}/parameters.json <<EOF
     "sync_retry_delay": "10_000ms",
     "sync_retry_nodes": 3,
     "prometheus_metrics": {
-        "socket_addr": "0.0.0.0:8010"
```

Comment:
👍 Nice catch!

Reply:
Thanks. In fact this json looks the same as `Docker/validators/parameters.json` so I changed to copy the file instead.